src/libstd/sys_common/poison.rs

```diff
@@ -60,10 +60,13 @@ pub struct Guard {
 
 /// A type of error which can be returned whenever a lock is acquired.
 ///
-/// Both Mutexes and RwLocks are poisoned whenever a thread fails while the lock
+/// Both [`Mutex`]es and [`RwLock`]s are poisoned whenever a thread fails while the lock
```

Comment:
Maybe you meant https://doc.rust-lang.org/std/sync/struct.Condvar.html#method.wait